source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/TextInputRenderer.java

```diff
@@ -187,76 +187,68 @@ public void afterTextChanged(Editable s)
-            }
-            else
-            {
-                textInputViewGroup = new LinearLayout(context);
-                textInputViewGroup.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT));
-                editText.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT, 1));
-                textInputViewGroup.addView(editText);
-
-                Resources.Theme theme = context.getTheme();
-                TypedValue buttonStyle = new TypedValue();
 
-                String url = action.GetIconUrl();
-                if(url != null && !url.isEmpty())
-                {
-                    ImageButton inlineButton = null;
-                    // check for style from resources
-                    if(theme.resolveAttribute(R.attr.adaptiveInlineActionImage, buttonStyle, true))
-                    {
-                        Context themedContext = new ContextThemeWrapper(context, R.style.adaptiveInlineActionImage);
-                        inlineButton = new ImageButton(themedContext, null, 0);
-                    }
-                    else
-                    {
-                        inlineButton = new ImageButton(context);
-                        inlineButton.setBackgroundColor(Color.TRANSPARENT);
-                        inlineButton.setPadding(16, 0, 0, 8);
-                    }
-
-                    InlineActionIconImageLoaderAsync imageLoader =
-                        new InlineActionIconImageLoaderAsync(
-                            renderedCard,
-                            inlineButton,
-                            url,
-                            editText
-                        );
-
-                    imageLoader.execute(url);
-                    textInputViewGroup.addView(inlineButton, new LinearLayout.LayoutParams(ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT, 0));
-                }
-                else
-                {
-                    String title =  action.GetTitle();
-                    Button inlineButton = null;
-                    // check for styles from references
-                    if(theme.resolveAttribute(R.attr.adaptiveInlineAction, buttonStyle, true))
-                    {
-                        Context themedContext = new ContextThemeWrapper(context, R.style.adaptiveInlineAction);
-                        inlineButton = new Button(themedContext, null, 0);
-                    }
-                    else
-                    {
-                        inlineButton = new Button(context);
-                        inlineButton.setBackgroundColor(Color.TRANSPARENT);
-                        inlineButton.setTextColor(Color.BLACK);
-                        inlineButton.setPadding(16, 0, 0, 8);
+        if(baseInputElement instanceof TextInput)
+        {
```

Comment:
should `{` go on this line, or at the end of the line above? inconsistent

Reply:
thanks

Reply:
thanks

---
In reply to: [246190275](https://github.com/Microsoft/AdaptiveCards/pull/2281#discussion_r246190275) [](ancestors = 246190275)